dotcom-rendering/src/components/marketing/epics/ContributionsLiveblogEpic.tsx

```diff
@@ -38,12 +38,34 @@ const getBackgroundColour = (tracking: Tracking) => {
 		tracking.abTestName.includes('_LB_EPIC_BG_COLOUR') &&
 		tracking.abTestVariant === 'VARIANT'
 	) {
-		return palette.neutral[100]; // COLOUR CHANGE TO GO HERE
+		return palette.brand[800];
 	} else {
 		return palette.neutral[100];
 	}
 };
 
+const getHeadingBackgroundColour = (tracking: Tracking) => {
+	if (
```

Comment:
This is all temporary, so really not a big deal, but you could reduce the duplication by checking this once in the component, and passing it through, e.g.
`const inColoursTestVariant = tracking.abTestName.includes('_LB_EPIC_BG_COLOUR') && tracking.abTestVariant === 'VARIANT';`

Reply:
We already pass another tracking variable in for the clientName to the css and there is limited scope for adding logic in the css, however, I've made a small refactor along the lines you suggest.